ArduCopter/config.h

```diff
@@ -168,8 +168,11 @@
  # define FS_EKF_THRESHOLD_DEFAULT      0.8f    // EKF failsafe's default compass and velocity variance threshold above which the EKF failsafe will be triggered
 #endif
 
-#ifndef EKF_ORIGIN_MAX_DIST_M
- # define EKF_ORIGIN_MAX_DIST_M         50000   // EKF origin and waypoints (including home) must be within 50km
+#ifndef EKF_ORIGIN_MAX_DIST_KM
+ # define EKF_ORIGIN_MAX_DIST_KM        250   // EKF origin and home must be within 250km
+#endif
+#ifndef EKF_ORIGIN_MAX_ALT_KM
+ # define EKF_ORIGIN_MAX_ALT_KM         10   // EKF origin and home must be within 10km vertically
```

Comment:
I think we can allow the altitude difference to be 50km.  The limitation was intended to be 50km (although it looks like a bug was introduced meaning it was only 500m) which was a conservative guestimate of when the accuracy started degrading.  I think with recent changes the guesimate has increased from about 100km to more like 800km.

Anyway, I think that a 10km altitude difference is concievable for example in the very unlikely case that a drone is dropped from an airplane.  Coincidentally I'm doing a test this month where the vehicle will be climb 10km and then land.  No plans to move home to 10km but it shows that 10km altitude is not impossible.